Marlin/src/gcode/calibrate/G76_M871.cpp

```diff
@@ -249,7 +249,7 @@ void GcodeSuite::G76() {
     bool timeout = false;
     for (;;) {
       // Move probe to probing point and wait for it to reach target temperature
-      do_blocking_move_to(noz_pos);
+      do_blocking_move_to(probe_noz_pos);
```

Comment:
Probe heating already takes a long time for me in the center of the bed. My safe homing point is in a corner so I think this would slow down the process a lot